src/index.js

```diff
@@ -1,20 +1,8 @@
-import {NativeModules, NativeEventEmitter, Platform} from 'react-native';
 import _ from 'underscore';
-
-const PLATFORM_ERROR_MESSAGE = Platform.select({ios: "- You have run 'pod install'\n", default: ''});
-const LINKING_ERROR = `The package 'react-native-key-command' doesn't seem to be linked. Make sure: \n\n
-${PLATFORM_ERROR_MESSAGE}\n
-- You rebuilt the app after installing the package\n
-- You are not using Expo managed workflow\n`;
-
-const KeyCommand = NativeModules.KeyCommand ? NativeModules.KeyCommand : new Proxy(
-    {},
-    {
-        get() {
-            throw new Error(LINKING_ERROR);
-        },
-    },
-);
+// eslint-disable-next-line import/extensions
+import getEventEmitter from './EventEmitter';
+// eslint-disable-next-line import/extensions
```

Comment:
**NOT A BLOCKER**: We can get away from these warnings if we rename `EventEmitter/index.web.js` & `KeyCommand/index.web.js` to `index.js`.

Reply:
@azimgd let's rename these files and remove the lint suppression.